src/Apps/Artwork/Components/OtherWorks/RelatedWorksArtworkGrid.tsx

```diff
@@ -75,22 +76,20 @@ class RelatedWorksArtworkGrid extends React.Component<
 
   render() {
     const {
-      artwork: {
-        layers,
-        layer: { artworksConnection },
-      },
+      artwork: { layers, layer },
       mediator,
     } = this.props
 
+    // The layer might have failed to fetch, so we use the `get` helper
+    // instead of ordinary destructuring.
+    const artworksConnection = get(layer, l => l.artworksConnection)
```

Comment:
:ceiling-cat: In cases like these I find `layer && layer.artworksConnection` to be more readable. `get` makes more sense imo for deeply nested access.